Extensions/Spine/spineruntimeobject-pixi-renderer.ts

```diff
@@ -193,7 +193,14 @@ namespace gdjs {
     }
 
     isAnimationComplete(): boolean {
-      return this._isAnimationComplete;
+      if (!isSpine(this._rendererObject)) {
```

Comment:
why is this check needed in a spine runtime renderer?

Reply:
Every methods of the renderer starts this way because this attribute can default on a Pixi object.

Reply:
ok 👍